layer.py

```diff
@@ -1,28 +1,25 @@
 import torch
 
 
-class LinearKernel(torch.nn.Module):
+class LinearKernel(nn.Module):
```

Comment:
The import needs to be changed. I'm only importing torch.

Reply:
I agree.